lib/ain-cpp-imports/src/bridge.rs

```diff
@@ -50,5 +50,6 @@ pub mod ffi {
         fn getNumCores() -> i32;
         fn getCORSAllowedOrigin() -> String;
         fn getNumConnections() -> i32;
+        fn getEvmCacheSizeLimit() -> usize;
```

Comment:
Would just call it as ECC pre-cache limit to side step this naming issue completely.

Reply:
Naming ECC pre-cache may not be accurate since the parameter isnt just used for signed tx cache, as it is also used for the validation cache.